<launch>

  <arg name="drone_id" default="0"/>

  <node pkg="swarm_bridge" name="drone_$(arg drone_id)_bridge_node_tcp" type="tcp_drone" output="screen">

    <param name="self_id" value="$(arg drone_id)"/>
    <param name="is_ground_station" value="false"/>

    <param name="drone_num" value="10"/>
    <param name="drone_ip_0" value="192.168.1.151" type="string"/>
    <param name="drone_ip_1" value="192.168.1.22" type="string"/>
    <param name="drone_ip_2" value="192.168.1.23" type="string"/>
    <param name="drone_ip_3" value="192.168.1.24" type="string"/>
    <param name="drone_ip_4" value="192.168.1.31" type="string"/>
    <param name="drone_ip_5" value="192.168.137.154" type="string"/>
    <param name="drone_ip_6" value="192.168.1.33" type="string"/>
    <param name="drone_ip_7" value="192.168.1.34" type="string"/>
    <param name="drone_ip_8" value="192.168.1.35" type="string"/>
    <param name="drone_ip_9" value="192.168.1.36" type="string"/>

    <param name="ground_station_num" value="1"/>
    <param name="ground_station_ip_0" value="192.168.137.1" type="string"/>

    <param name="pub_occupancy" value="true" type="bool"/>

  </node>
</launch>
